tests/admin_docs/tests.py

```diff
@@ -463,3 +456,20 @@ def test_initial_header_level(self):
         header = 'should be h3...\n\nHeader\n------\n'
         output = utils.parse_rst(header, 'header')
         self.assertIn('<h3>Header</h3>', output)
+
+
+class AdminDocViewFunctionsTests(SimpleTestCase):
```

Comment:
Please rebase the tests after 231743414710c60090728e86f33d81cbc43df1d1 and add the test case from https://github.com/django/django/pull/7135, unless you think it's redundant.